src/mon/PGMap.cc

```diff
@@ -143,6 +149,14 @@ void PGMapDigest::dump(ceph::Formatter *f) const
     f->close_section();
   }
   f->close_section();
+  f->open_array_section("pool_pg_unavailable_map");
```

Comment:
do you think we can leave only one array section instead of opening a new section for each iterator?